app/code/Magento/Catalog/Controller/Adminhtml/Category/Add.php

```diff
@@ -39,16 +46,16 @@ public function __construct(
     /**
      * Add new category form
      *
-     * @return \Magento\Backend\Model\View\Result\Forward
+     * @return ResultInterface
      */
-    public function execute()
+    public function execute(): ResultInterface
```

Comment:
I don't think it is backward compatible.

Reply:
> I don't think it is backward compatible.

There were no complains about backward incompatible change from static tests, usually we do have

Reply:
Hm. @sidolov, remember you mentioned in some of PRs adding return type even for controller action is not compatible? Do we have all BC violations reported by automated tests now?